### DISCUSSION TITLE
What to put in gitignore?

### DISCUSSION THREAD

**Discussion by AppleFritter58** [Vinzent03/obsidian-git#231] [Q&A]
I am concerned with what to put in gitignore, because I don't want to remove too much sync capabilities.

Here is my current file (I use Mac, Windows, and iOS).

```# MacOS settings and caches
.trash/
.DS_Store

# Laptops
.obsidian/workspace

# Mobile
.obsidian.mobile/workspace

**Comment by Vinzent03** [comment]
I don't know how often you change Obsidian's settings, but I think your current config is enough.

**Reply by bkbelton** [reply]
What is the proper way to sync a vault's plugins and plugin settings across 2 identical (M1 ventura) macs? I removed .obsidian from .gitignore thinking that would then sync everything, but when cloning the vault I found that I had to reinstall plugins.Once they were re-enabled, some, but not all, of the plugins seemed to pick up the customized settings, although I haven't tested enough to know if that is actually true. I would like to sync plugin activation and settings if possible. This is my .gitignore:

```gitignore
# to exclude Obsidian's settings (including plugin and hotkey configurations)
.obsidian.mobile
# OR only to exclude workspace cache
.obsidian/workspace.json
# Add below lines to exclude OS settings and caches
.trash/
**//.DS_Store
```

Thank you.

**Reply by Vinzent03** [reply]
I don't know what `.obsidian.mobile` should be. Some plugins, including this plugin, store some config in the actual vault and not in `.obsidian`. By only ignoring `.obsidian/workspace.json` your plugins activation and actual plugins should be committed.
